src/resources/bdfdCreation.md

```diff
@@ -0,0 +1,17 @@
+# BDFD's Creation
+*How Bot Designer For Discord became what it is today.*
+
+### Who Develops BDFD?
+| Name | Discord Tag | Position 
+| ------------- | ------------- | -------------
+Jakub Tomana | wiadr000#1947 (kubastick) | Owner/Developer
+Mikołaj Radkowski  | noituri#2489 | Developer/DevOp
+Bartłomiej Skoczeń | MineBartekSA#4899 | Developer/DevOp
+
+### The Company
+*[Nilpointer Software](https://nilpointer.software/)* is the parent company of *Bot Designer For Discord*. *Nilpointer Software* a start-up focused on providing fast and quality software.
```

Comment:
It should be Nil**P**ointer Software instead of Nil**p**ointer Software

Reply:
Resolved 👍